staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -198,7 +198,15 @@ func validateBackendMode(pluginName string, mode string) error {
 	return fmt.Errorf("invalid audit %s mode %s, allowed modes are %q", pluginName, mode, strings.Join(AllowedModes, ","))
 }
 
-func validateBackendBatchConfig(pluginName string, config pluginbuffered.BatchConfig) error {
+func validateBackendBatchOptions(pluginName string, options AuditBatchOptions) error {
+	if err := validateBackendMode(pluginName, options.Mode); err != nil {
+		return err
+	}
+	if options.Mode != ModeBatch {
+		// Don't validate the unused options.
+		return nil
+	}
+	config := options.BatchConfig
```

Comment:
What will happen if --audit-log-batch-max-wait is set to zero?

Reply:
It will hotloop. Not great, but not exactly invalid. I'd prefer to leave it as-is here.